pkg/services/hivelocity/client/mock/mock_client.go

```diff
@@ -100,6 +100,18 @@ var NoTagsDevice = hv.BareMetalDevice{
 	OsName:      defaultImage,
 }
 
+// MultiTagsDevice is a device which has no tags.
+var MultiTagsDevice = hv.BareMetalDevice{
+	Hostname: "host3-unused",
+	Tags: []string{
+		"caphvlabel:" + hvtag.DeviceTag{Key: hvtag.DeviceTagKeyDeviceType, Value: "hvCustom"}.ToString(),
```

Comment:
I think you shouldn't mix the labels starting with "caphvlabel:" and the device tags that are pre-defined.